jenkins/Jenkinsfile.linux.master

```diff
@@ -30,7 +30,7 @@ pipeline {
                 S3_BUCKET_NAME = 'REDACTED'
             }
             steps {
-                sh 'npm run jenkins:build:linux'
+                sh 'yarn run jenkins:build:linux'
```

Comment:
Not blocking the PR with this at all, but you can omit `run` so you just have `yarn jenkins:build:linux`